Post about Guanyang joining the project

hlapp opened the pull request.

GZhang2 commented on _posts/blog/2017-05-05-welcoming-guanyang.md:
```diff
@@ -0,0 +1,49 @@
+[ARBOR]). This search has recently concluded, and we are delighted
+to welcome Dr. [Guanyang Zhang] to our team.
+
+Guanyang is a taxonomist and an evolutionary biologist. He obtained a
```
Suggested wording: Guanyang is a taxonomist and an entomologist.

GZhang2 commented on _posts/blog/2017-05-05-welcoming-guanyang.md:
```diff
@@ -0,0 +1,49 @@
+associations, taxonomic concepts and environmental ontology.
+
+Throughout his research, Guanyang found an imminent need to augment
+the precision and universality when communicating taxonomic concepts,
```
... augment the precision and universality *of taxonomy for communicating and referencing biological information.*

GZhang2 commented on _posts/blog/2017-05-05-welcoming-guanyang.md:
```diff
@@ -0,0 +1,49 @@
+
+Throughout his research, Guanyang found an imminent need to augment
+the precision and universality when communicating taxonomic concepts,
+which is why he felt the phyloreferencing project as a natural fit for
```
.. *is* a natural fit to his training and research objectives.

GZhang2 commented on _posts/blog/2017-05-05-welcoming-guanyang.md:
```diff
@@ -0,0 +1,49 @@
+Throughout his research, Guanyang found an imminent need to augment
+the precision and universality when communicating taxonomic concepts,
+which is why he felt the phyloreferencing project as a natural fit for
+expanding his training. He will be particularly interested in
```
... particularly *involved* in ..

GZhang2 commented on _posts/blog/2017-05-05-welcoming-guanyang.md:
```diff
@@ -0,0 +1,49 @@
+expanding his training. He will be particularly interested in
+developing and testing testing phyloreferences for insects and bacteria.
+
+Guanyang already participated in the [face-to-face project meeting]({%
```
delete *already*

hlapp commented:
@GZhang2 I made the changes as requested. OK to publish?

GZhang2 commented:
Thanks Hilmar. I was just going to do the edit myself. Took me a while to
find out how to edit. It looks good to go. Thanks!!

Guanyang

Guanyang Zhang, PhD
Postdoc Research Associate @ The Franz Lab
School of Life Sciences, Arizona State University
PO Box 874501 (mail) Room# LSA129, 427 East Tyler Mall (package)
Tempe, AZ 85287
somanyinsects.org
Email: gyz151@gmail.com
Tel: 480-965-2850 (office) 812-320-2619 (mobile)

On Thu, May 4, 2017 at 7:36 PM, Hilmar Lapp <notifications@github.com>
wrote:

> @GZhang2 <https://github.com/GZhang2> I made the changes as requested. OK
> to publish?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/phyloref/phyloref.github.io/pull/18#issuecomment-299357476>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKlMUqiF6mdmo2YEbkYFsyb1AOD2ypNXks5r2otLgaJpZM4NRYcg>
> .
>

gaurav commented:
This isn't up at http://www.phyloref.org/blog/ -- I think because it was technically published in the future (May 5) when the update was made. We could add in an extra, empty commit to trigger a rebuild: http://stackoverflow.com/a/24099328/27310

hlapp commented:
I squashed the two commits into one and force pushed, but as suspected that didn't change anything. The future dated-post published fine locally, so it would have been a surprise if it didn't on Github.

The problem is that the site build fails at Github, but the only error message we're getting is this:
```
Page build failed. For more information, see https://help.github.com/articles/troubleshooting-github-pages-builds/.
```
I've now updated the Jekyll container image I'm using for local deployment (see `Dockerfile` and `serve-local.sh`), and now I am indeed getting an error:
```
Deprecation: A call to '{{ post_url 2016-07-20-welcoming-gaurav }}' did not match a post using the new matching method of checking name (path-date-slug) equality. Please make sure that you change this tag to match the post's name exactly.
       Deprecation: A call to '{{ post_url 2016-10-05-postdoc2-ad }}' did not match a post using the new matching method of checking name (path-date-slug) equality. Please make sure that you change this tag to match the post's name exactly.
  Liquid Exception: Could not find post "2017-04-26-first-face-to-face-project-meeting" in tag 'post_url'. Make sure the post exists and the name is correct. in /srv/jekyll/_posts/blog/2017-05-05-welcoming-guanyang.md
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Could not find post "2017-04-26-first-face-to-face-project-meeting" in tag 'post_url'. Make sure the post exists and the name is correct.
```
I'll try and fix that.
